docs/src/components/layouts/utility-page-layout/__tests__/utility-class-table-factory.test.js.bad

```diff
@@ -0,0 +1,24 @@
+import { render, screen } from '@testing-library/react';
```

Comment:
For this file I would like you to:
1. Remove .bad from the file name.
2. Run `jest` in docs
3. Create a new issue with the error output AND the contents of this file.
4. Delete the file after the issue has been created.